benchmark/serial/centrality.jl

```diff
@@ -0,0 +1,37 @@
+SUITE["centrality"] = BenchmarkGroup(
+    [], "graphs" => BenchmarkGroup([]), "digraphs" => BenchmarkGroup([])
+)
+
+# graphs
+SUITE["centrality"]["graphs"]["degree_centrality"] = @benchmarkable [
+    Graphs.degree_centrality(g) for (_, g) in $GRAPHS
```

Comment:
it's weird to benchmark all graphs inside of the same loop, but let's not change the contents of the benchmark for now

Reply:
yes. I didn't want to tweak anything. Once the foundation is there improvements on the content will be the next thing.